linera-core/src/unit_tests/worker_tests.rs

```diff
@@ -3560,7 +3570,11 @@ where
     let public_key = signer.generate_new();
     let mut env = TestEnvironment::new(storage, false, false).await;
     let balance = Amount::from_tokens(5);
-    let chain_1_desc = env.add_root_chain(1, public_key.into(), balance).await;
+    let mut ownership = ChainOwnership::single(public_key.into());
+    ownership.timeout_config.fallback_duration = TimeDelta::from_secs(5);
```

Comment:
It'd be worth adding a comment why we set the fallback manually and why to 5 seconds.

Reply:
Done in https://github.com/linera-io/linera-protocol/pull/3905/commits/17661636e0526aa53a83cea225f20c2eecd65272.